test_unstructured/partition/html/test_partition.py

```diff
@@ -30,7 +30,7 @@
     Title,
 )
 from unstructured.file_utils.encoding import read_txt_file
-from unstructured.partition.html import HtmlPartitionerOptions, partition_html
+from unstructured.partition.html.partition import HtmlPartitionerOptions, partition_html
```

Comment:
Can this import statement be `from unstructured.partition.html import HtmlPartitionerOptions, partition_html` since we have the line `__all__ = ["HtmlPartitionerOptions", "partition_html"]` in `unstructured/partition/html/__init__.py`?

Reply:
It could be. Actually, thinking about it for a minute, I think it would be better that way actually.

The import `from unstructured.partition.html import partition_html` is the one we expect end users to use. Adding the actual location is an implementation detail that is subject to change. I'll change it for `partition_html`, that way if importing from `...partition.html` stopped working for some reason (which would break user code) then the tests would let us know.

On `HtmlPartitionerOptions` I think I'll leave that in the explicit location import and remove it from `__init__.py` since we don't have any users for that yet and better not to over-publish.